public/index.html

```diff
@@ -25,6 +25,9 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Readaholic</title>
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Libre+Baskerville:wght@700&family=Montserrat:wght@400;600&display=swap');
```

Comment:
Maybe you need to double check whether your Storybook is actually making use of this font or not. If I recall correctly, using `@import` in the index.html did not work for me.

Instead I created a dedicated `Montserrat.css` file, imported the font there and then, in my GlobalStyles.js imported said css file.

Reply:
Ok! I just followed the alert in the console which said that I should not make the import in the GlobalStyles.js and should use index.html instead. 🙈  Checked storybook and it works. But will keep this in mind if I have problems with my fonts and storybook in the next iterations. 👍